### DISCUSSION TITLE
GRAFFER

### DISCUSSION THREAD

**Discussion by jtappin** [gnudatalanguage/gdl#1111] [Show and tell]
Since most commonly used features of GRAFFER are now working under both GDL & IDL, I have pushed it to GitHub (the old U. of Bham site is long gone).

[GDL version of Graffer](https://github.com/jtappin/graffer-gdl)

**Comment by alaingdl** [comment]
Great !
Maybe you can ask @olebole to add that into the Debian Astro and in the [sub-category GDL](https://salsa.debian.org/search?utf8=%E2%9C%93&search=gdl&group_id=2133&project_id=&scope=&search_code=false&snippets=false&repository_ref=&nav_source=navbar) in it ?

**Comment by GillesDuvert** [comment]
Hi, could we use your cw_pdmenu_plus to mimic IDL's cw_pdmenu?

**Reply by jtappin** [reply]
Both cw_pdmenu_plus and cw_enter, could potentially be cut down to be versions of cw_pdmenu and cw_field respectively, so in principle: yes.
However, I don't think either is a strict superset of the equivalents, and also at a distance of 25 years I don't recall whether they were directly based on early versions of the IDL library or not.

**Reply by GillesDuvert** [reply]
cw_pdtmenu (yes apparently I have a very old version of graffer) was indeed just an (improved) copy of the copyrighted IDL one.
But I think cw_pdmenu_plus is alphabetically way different and (of course the purpose being the same, the way to achieve this purpose using a well defined command langage cannot be very different!) can be IMHO proposed under a gpl or equivalent license.

**Reply by jtappin** [reply]
Feel free to use anything from Graffer in GDL if it looks like it will be useful.
